### DISCUSSION TITLE
Update to the website failing after adding new entry to papers.bib

### DISCUSSION THREAD

**Discussion by sai-soum** [alshedivat/al-folio#2567] [Q&A] [answered]
[
![Screenshot 2024-07-11 at 20 36 33](https://github.com/alshedivat/al-folio/assets/79447355/3142c811-3e97-46e2-8239-ab5e4c030164)
](url)

**Comment by george-gca** [comment, marked as answer]
Check our [FAQ](https://github.com/alshedivat/al-folio/blob/master/FAQ.md#my-code-runs-fine-locally-but-when-i-create-a-commit-and-submit-it-it-fails-with-prettier-code-formatter-workflow-run-failed-for-master-branch-how-do-i-fix-that).
